src/Tagger.Git/GitTagger.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.IO;
+using System.Linq;
+using LibGit2Sharp;
+using System.Collections.Generic;
+
+namespace Microsoft.Omex.Log.Tagger.Git
+{
+	internal class GitTagger : ITagger
+	{
+		protected string m_repoRoot;
+
+		protected string m_pathSeparator;
+
+		protected string m_commitId;
+
+		protected string m_urlTemplate;
+
+		private readonly ITagCache m_cache;
```

Comment:
>private [](start = 2, length = 7)

A general question on the coding style for this project. It does not adhere to the style in Gating, is that intentional?